test/e2e/allocation.new.test.js

```diff
@@ -0,0 +1,52 @@
+import { Selector, t } from 'testcafe'
+
+import { pmuUser } from './_roles'
+import { allocations, newAllocation } from './_routes'
+import { allocationDetails, allocationCriteria, page } from './pages/'
+
+const selectors = {
+  cancelJourney: {
+    linkToFirstAllocation: Selector('.govuk-table td:first-of-type a'),
+    cancelLink: Selector('.app-link--destructive').withText(
+      'Cancel allocation'
+    ),
+    statusHeading: Selector(
+      '.app-message.app-message--temporary .app-message__heading'
+    ).withText('Allocation cancelled'),
+  },
+}
+
+async function createAllocation() {
+  await allocationDetails.fill()
+  await page.submitForm()
+  await allocationCriteria.fill()
+  await page.submitForm()
+  return t
+    .expect(page.getCurrentUrl())
+    .match(/\/allocation\/[\w]{8}(-[\w]{4}){3}-[\w]{12}\/confirmation$/)
+}
+
+fixture('New PMU allocation').beforeEach(async t => {
+  await t.useRole(pmuUser).navigateTo(newAllocation)
+})
+test('by default', t => {
+  return createAllocation()
+})
+
+fixture('Cancel allocation').beforeEach(async t => {
+  await t.useRole(pmuUser).navigateTo(newAllocation)
+  await createAllocation()
+  await t.useRole(pmuUser).navigateTo(allocations)
+})
+test('Cancel allocation', async t => {
+  await t.click(selectors.cancelJourney.linkToFirstAllocation)
```

Comment:
Can we also move these selectors to a page object. We can have a separate cancel-allocation page object like we have for [cancel-move](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/3a64f5b2d6669722908c86c41cd87e9ca7b78e9d/test/e2e/pages/cancel-move.js#L1)